packages/treasury/src/interest.js

```diff
@@ -48,7 +48,7 @@ export function makeInterestCalculator(
     let growingDebt = newDebt;
     while (newRecent + chargingPeriod <= currentTime) {
       newRecent += chargingPeriod;
-      const newInterest = multiplyBy(growingDebt, ratePerChargingPeriod);
+      const newInterest = ceilMultiplyBy(growingDebt, ratePerChargingPeriod);
```

Comment:
I'm confused... wasn't this change in an earlier PR?

Reply:
yep. rebased.